pkg/cloud/ibmcloud/clients/machines.go

```diff
@@ -163,7 +163,7 @@ func (gs *GuestService) GuestDelete(Id int) error {
 	return err
 }
 
-func (gs *GuestService) GuestList() ([]datatypes.Virtual_Guest, error) {
+func (gs *GuestService) listGuest() ([]datatypes.Virtual_Guest, error) {
```

Comment:
make this ListGuest, it might be used somewhere later elsewhere...

Reply:
Is this a perdefined interface? 
I'd like to keep it as is if not. We can keep a function scope limited unless it is really used by others.

Reply:
We can update it to `ListGuest` when this is requested ;-)